lib/s3botoutils.js

```diff
@@ -10,7 +10,7 @@ import {
 import { config } from "../config.js";
 import * as Stream from "stream";
 
-const s3 = new S3Client({ region: config["AWS_REGION"] });
+const s3 = new S3Client({ region: config["REGION"] });
```

Comment:
you can get it as `config.REGION` , need not to dynamically compute the property here.

Reply:
This has been addressed